### ISSUE TITLE
Problem when sending chunked messages to an endpoint

### ISSUE THREAD

**Issue by saschadoemer** [DKE-Data/agrirouter-sdk-java#182]
**Describe the bug**
A chunked message is not correctly recognized by the I/O Tool.

**Additional context**
See also: https://agrirouter.atlassian.net/browse/AGRISP-1576

**Comment by saschadoemer** [comment]
There was a problem with the implementation. I fixed the chunking mechanism and the files are shown correctly within the I/O-Tool now.

Before sending the file via `SendChunkedMessageTest`.

![image](https://user-images.githubusercontent.com/1309344/156769034-1b4a2424-01e1-4794-9751-55e57cbfa76d.png)

After sending the file via `SendChunkedMessageTest`.

![image](https://user-images.githubusercontent.com/1309344/156769290-083e175d-0320-4a9c-ba70-98b638af8399.png)

The message did arrive without any difficulty. Checked this with only with task data to have a simple test case. Test case is all good.

![image](https://user-images.githubusercontent.com/1309344/156769454-4e367acd-ff49-459b-8ef8-4074c90b0dac.png)
